README.md

```diff
@@ -42,6 +42,7 @@ All documents can in general be found in the doc folder.
 - Use WebStorm (or Git knowledge) to clone the repo to a local folder
 - Run `npm install`
 - Run `npm run-script travis` to check whether everything works correctly
+- Run `git submodule init && git submodule update -recursive` to make the analytics work
```

Comment:
Might not be neccassary right now, but it doesn't hurt and might be needed in the future